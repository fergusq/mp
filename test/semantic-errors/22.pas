program p;
procedure f(i:integer);
begin writeln(1)end;
begin f(2.0)end.
