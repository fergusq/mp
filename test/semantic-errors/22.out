Semantic error: Type mismatch: wrong argument type for param i of function f: expected integer, got real
